src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -36,7 +36,7 @@
         display: flex;
         flex-flow: column nowrap;
         // Firefox needs this
-        min-height: 0;
+        min-height: $clr_baselineRem_6;
```

Comment:
Wait, what? Why are we making the table part of the datagrid at least 6 baselines high? That's far more than the visual spec.